schema.ts

```diff
@@ -1371,11 +1425,11 @@ export function normalizeSchema<T extends Schema>(schema: T): T {
       description: schema.description,
       attribution: schema.attribution,
       includeUnknownProperties: schema.includeUnknownProperties,
-      titleProperty: titleProperty ? normalizeSchemaKey(titleProperty) : undefined,
-      subtitleProperties: subtitleProperties ? subtitleProperties.map(normalizeSchemaKey) : undefined,
-      imageProperty: imageProperty ? normalizeSchemaKey(imageProperty) : undefined,
-      descriptionProperty: descriptionProperty ? normalizeSchemaKey(descriptionProperty) : undefined,
-      linkProperty: linkProperty ? normalizeSchemaKey(linkProperty) : undefined,
+      titleProperty: titleProperty ? tryNormalizeSchemaPropertyType(titleProperty) : undefined,
```

Comment:
So just confirming that we're omitting `idProperty` and `featuredProperties` from allowing paths? I think for consistency it would probably make people's intuition if these supported it, but it requires code changes and may break cross-doc, TypeForm and other dynamic schema situations where the featured columns could resemble paths, so I guess safer this way. @ekoleda-codaio WDYT?

Reply:
Yeah called out in the PR but I left a todo to go enable this but after starting down that path realized there was a lot more functionality required to enable that, so didn't want to muddle the waters currently

Reply:
The way the pathing logic is implemented I don't think it should break those, since it will always look for an exact match first. It'll just be wonky if you try to extend it to do anything complicated (which I don't imagine people will do for dynamic schemas given there should inherently be no guarantees on what properties are present?)